go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/submariner-io/subctl
 
-go 1.19
+go 1.18
```

Comment:
Please don’t revert to 1.18.

Reply:
I did not. Don't know how this ended up changing.